Tasks/AzureCLIV1/task.json

```diff
@@ -18,7 +18,7 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 132,
+        "Minor": 136,
```

Comment:
Should it be 137 or you plan to port this to 136?

Reply:
No, changed it to 137 👍